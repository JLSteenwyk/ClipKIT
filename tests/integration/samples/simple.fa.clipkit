>1
A-GAT
>2
A-GAT
>3
A-GTA
>4
AGATA
>5
ACaT-
