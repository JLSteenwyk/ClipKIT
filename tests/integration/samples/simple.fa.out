>1
A-TA
>2
A--A
>3
A--T
>4
AG-T
>5
AC-T
